packages/playwright-core/src/server/registry/download.ts

```diff
@@ -88,6 +88,7 @@ export async function download(
     log(
         `downloading ${progressBarName} - attempt #${attempt}`
     );
+    const url = Array.isArray(urls) ? urls[(attempt - 1) % urls.length] : urls;
```

Comment:
As is, if our primary is down, all our clients will fallback to Verizon. Is that what we want, or should the fallback order be randomized so the fallbacks (assuming they themselves are not down), see partial load?